TESTING.md

```diff
@@ -1,41 +1,28 @@
 # APM Server-Testing
 
 ## Automated Testing
-To run the full testsuite of APM Server, run:
+The tests are built on top of the [Beats Test Framework](https://github.com/elastic/beats/blob/master/docs/devguide/testing.asciidoc), where you can find a detailled description on how to run the Testsuite.
+
+### Quick Overview
+Run the full testsuite of APM Server, which needs all dependencies to run locally or in a Docker environment:
```

Comment:
I kept it very generic since any external dependencies could be required, and being more specific here would probably be the first thing getting outdated. The detailed description should be provided within the _beats_ project which owns the test framework code.